src/sinks/http.rs

```diff
@@ -118,6 +118,12 @@ fn http(config: HttpSinkConfig, acker: Acker) -> Result<super::RouterSink, Strin
         HttpRetryLogic,
     );
 
+    if !verify {
+        warn!(
+            message = "Turning off verify_certificate in http sink can introduce security vulnerabilities"
```

Comment:
I'd actually say something like `'verify_certificate' is DISABLED this may lead to security vulnerabilities.`

Reply:
Done